charts/privatebin/values.yaml

```diff
@@ -43,8 +43,8 @@ controller:
 
 initChownData:
   image:
-    repository: busybox
-    tag: 1.31.1
+    repository: privatebin/chown
+    tag: 1.34.1-musl-1.2.2-r3
     pullPolicy: IfNotPresent
```

Comment:
We shouldn't need this at all anymore, since we no longer chown the files.

Reply:
Understood, I removed the init container section in commit 5fdc1d4

Reply:
I'm going to drop this section from `values.yaml` as well, and then merge.